### DISCUSSION TITLE
Edit Cell Action Menu customization need supported

### DISCUSSION THREAD

**Discussion by HitEagle** [KevinVandy/material-react-table#983] [Q&A]
From the code, there's no other menu passed from out-of-box, so the edit cell action menu cannot be customized.
Any ideas or snippet to make this work is very appreciated. :)
https://github.com/KevinVandy/material-react-table/blob/v2/packages/material-react-table/src/components/table/MRT_TableContainer.tsx#L108

**Comment by KevinVandy** [comment]
What's your use case for customizing the menu component instead of the menu items themselves that you can't already do with CSS?

**Comment by HitEagle** [comment]
Thanks for the answer, Kevin. I will try to override its css, will close the thread until meet the use case that need to customizing the cell action menu component.
